tlatools/org.lamport.tlatools/src/tlc2/debug/TLCStackFrame.java

```diff
@@ -669,6 +669,8 @@ public StoppedEventArguments getStoppedEventArgument() {
 			// the term "exception" appears in the front-end.
 			eventArguments.setReason(StoppedEventArgumentsReason.EXCEPTION);
 			eventArguments.setText(this.exception.getMessage().replaceAll("(?m)^@!@!@.*", ""));
+		} else {
+			eventArguments.setReason(StoppedEventArgumentsReason.STEP);
```

Comment:
Why `STEP` and not `PAUSE`, or `BREAKPOINT`, ...?  Does the [spec](https://microsoft.github.io/debug-adapter-protocol/specification) mandate `STEP` as a default?

Reply:
That was just because I found reason-missing SEA here when testing step-over, without decent intention.
The spec doesn't mention about default value so using empty string sounds more appropriate. Let me fix.